src/Mod/Draft/DraftGeomUtils.py

```diff
@@ -1346,7 +1346,7 @@ def offsetWire(wire,dvec,bind=False,occ=False,widthList=None, offsetMode=None, a
         if i == 0:
             if alignListC[0] == 'Center':
                 delta = DraftVecUtils.scaleTo(delta, delta.Length/2)
-            #No need to do anything for 'Left' and 'Rigtht' as orginal dvec have set both the direction and amount of offset correct
+            #No need to do anything for 'Left' and 'Rigtht' as original dvec have set both the direction and amount of offset correct
```

Comment:
@luzpaz LOL. I just noticed that you missed the word `Rigtht`.

Reply:
'Typo saturation' :lol:

Reply:
I can't atm but can someone make a PR to the codespell dictionary?